DER/DerivativesContracts/Options.rdf

```diff
@@ -594,23 +469,40 @@
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">An exercise schedule may be as simple as a single date or date period. However, in more complex cases, it may be an ad hoc schedule of individual dates, or a regular schedule of periodic exercise dates.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:ObjectProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-der-der-opt;hasExerciseStyle">
+	<owl:ObjectProperty rdf:about="&fibo-der-drc-opt;hasExerciseStyle">
 		<rdfs:subPropertyOf rdf:resource="&lcc-cr;isClassifiedBy"/>
 		<rdfs:label>has exercise style</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;Option"/>
 		<rdfs:range rdf:resource="&fibo-sec-dbt-ex;ExerciseConvention"/>
 		<skos:definition>indicates the exercise convention specified for the option</skos:definition>
 	</owl:ObjectProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-der-der-opt;hasRelativePaymentDate">
+	<owl:ObjectProperty rdf:about="&fibo-der-drc-opt;hasOptionHolder">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-agr-ctr;hasContractParty"/>
+		<rdfs:label>has option holder</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;Option"/>
+		<rdfs:range rdf:resource="&fibo-der-drc-opt;OptionHolder"/>
+		<skos:definition>indicates the owner of the option</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-der-drc-opt;hasOptionWriter">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-agr-ctr;hasContractParty"/>
+		<rdfs:label>has option writer</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;Option"/>
+		<rdfs:range rdf:resource="&fibo-der-drc-opt;OptionIssuer"/>
+		<skos:definition>indicates the original issuer of the option</skos:definition>
```

Comment:
not sure about "original" here since there can only be one issuer.

Reply:
@rivettp - that's what I heard during our working session, but I've removed the word 'original'